### PR TITLE
Fix item creator text alignment to start from bottom

### PR DESCRIPTION
**Pull request by dualcnhq**

### CODE REVIEW

File: src/tiles/grid/item-tile.ts [hunk 1]
```diff
@@ -223,19 +218,21 @@ export class ItemTile extends LitElement {
...
         -webkit-box-orient: vertical;
+        display: flex;
+        justify-content: center;
+        align-items: flex-end;
+        padding-top: 1rem;
       }
 
-      #text-by {
+      .item-creator span:before {
+        content: 'By: ';
```
Comment (jbuckner) [inline src/tiles/grid/item-tile.ts:235]: This feels wrong to put semantically meaningful content in your CSS. What effect are you trying to achieve with this?

Comment (dualcnhq) [reply src/tiles/grid/item-tile.ts:235]: i tried to separate the span text but got different positions on each so i did it this way.

is it better if i change it to: `<span><strong>By:</strong>${itemCreator}</span>`

Comment (jbuckner) [reply src/tiles/grid/item-tile.ts:235]: Yes, that would be better

Comment (dualcnhq) [reply src/tiles/grid/item-tile.ts]: updated...

File: src/tiles/grid/item-tile.ts [hunk 2]
```diff
@@ -223,19 +218,20 @@ export class ItemTile extends LitElement {
...
         -webkit-line-clamp: 2;
         -webkit-box-orient: vertical;
+        display: flex;
+        justify-content: center;
+        align-items: flex-end;
+        padding-top: 1rem;
       }
 
-      #text-by {
+      #by {
```
Comment (jbuckner) [inline src/tiles/grid/item-tile.ts:234]: Does this `#by` element exist? I didn't notice it above.

Comment (dualcnhq) [reply src/tiles/grid/item-tile.ts]: good catch... i just removed it

**Review by jbuckner — APPROVED** [review]
Looks good! 👍

**Pull request merged**